CIME/case/case_submit.py

```diff
@@ -51,12 +51,7 @@ def _submit(
 
     # Check if CONTINUE_RUN value makes sense
     # if submitted with a prereq don't do this check
-    if (
-        job != "case.test"
-        and case.get_value("CONTINUE_RUN")
-        and hasMediator
-        and not prereq
-    ):
+    if case.get_value("CONTINUE_RUN") and hasMediator and not prereq:
```

Comment:
Was it intentional that you built this on top of the changes in #4345? It looks like your fix in #4345 has evolved from this, so this may cause problems with the merge.

Reply:
Oh, I see you just closed #4345 . So then I guess the question becomes which one you want to use: this one that applies this code to all test cases, or what you had in #4345, which had exceptions for PET and NCK. If the latter, can you add a comment about why you need an exception for those test types (if you know)?

Reply:
I thought that an exception was needed then retested and didn't need an exception, currently I have removed all exceptions and would like to pass it off to someone else to test independently.

Reply:
Sounds good, thanks.